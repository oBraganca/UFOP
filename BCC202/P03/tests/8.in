50
I Arthur
840591 gol 2 2022 Bernardo 205
148016 tcross 5 2002 Gabriel 209
384953 fox 5 2003 Pedro 206
924730 ferrari 8 2014 Isabella 208
907130 corolla 4 1985 Gabriel 209
410280 yars 6 2001 Bernardo 205
109591 yars 3 1997 Pedro 206
717639 tcross 2 1981 Davi 207
970952 ferrari 5 2021 Isabella 208
723764 yars 8 1996 Pedro 206
327846 corolla 1 1982 Arthur 203
654157 gol 1 1985 Miguel 201
633536 hb20 7 1980 Davi 207
746207 pulse 8 1987 Gabriel 209
196852 palio 2 2011 Davi 207
304571 onix 3 2017 Helena 204
709668 corolla 9 1981 Bernardo 205
812130 hb20 10 2012 Sophia 202
899619 corolla 4 2022 Arthur 203
981629 onix 9 1987 Arthur 203
209836 creta 5 2004 Helena 204
980446 fox 4 1997 Helena 204
485650 tcross 7 2004 Bernardo 205
776079 pulse 5 2001 Davi 207
940778 creta 4 2005 Isabella 208
148199 yars 3 2000 Alice 200
135620 corolla 2 2000 Sophia 202
450083 fox 4 2004 Alice 200
996508 pulse 6 2007 Helena 204
379174 corolla 7 1988 Helena 204
435834 ferrari 4 2019 Miguel 201
111420 onix 7 1982 Sophia 202
859096 argo 5 1997 Pedro 206
815681 argo 7 2011 Alice 200
741785 corolla 8 2020 Gabriel 209
781725 argo 6 2003 Sophia 202
456558 argo 4 2006 Alice 200
207504 mobi 2 2003 Helena 204
343068 gol 1 2009 Davi 207
665490 ferrari 2 1997 Davi 207
873418 ferrari 3 2007 Alice 200
883819 nivus 3 2014 Gabriel 209
270244 onix 2 1986 Isabella 208
148632 polo 3 1980 Helena 204
619709 nivus 8 2014 Sophia 202
901421 pulse 3 1983 Sophia 202
980620 tcross 10 2006 Bernardo 205
648476 creta 7 1989 Pedro 206
562255 nivus 6 2023 Gabriel 209
182975 pulse 8 1985 Alice 200
