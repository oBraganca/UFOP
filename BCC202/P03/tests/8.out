onix [1987|9|981629]
corolla [2022|4|899619]
corolla [1982|1|327846]
